pkg/reconciler/instances/eventing/preaction/preaction.go

```diff
@@ -0,0 +1,42 @@
+package preaction
```

Comment:
The code in this file and for `postaction` look almost identical, is this duplication meaningful?
Might be better to have one step looping logic and concentrate the differences on the specific steps + parametric initialisation.

Reply:
You are right. Changed.